### DISCUSSION TITLE
MultiSelect with the number of selected elements

### DISCUSSION THREAD

**Discussion by iakovalgaev** [mantinedev/mantine#2156] [Feature requests]
I really want to be able to set a border, after which not a label will be displayed, but "{count} items selected". Perhaps it would be great to still be able to control this text. For example, "_{count} countries selected_"
![image](https://user-images.githubusercontent.com/61377069/185375283-e4c78861-b0d2-416f-a5bb-af1958fd96a3.png)

**Comment by iakovalgaev** [comment]
Is it planned to release this?

**Reply by rtivital** [reply]
Currently, it is not planned to be added at least until 7.0 release

**Comment by 0xrasla** [comment]
i really need this!

![image](https://github.com/mantinedev/mantine/assets/76244740/6b381027-0e5a-47d5-88e2-ab57c9b17739)
